integrationtest/src/test/java/dk/kvalitetsit/itukt/integrationtest/api/ManagementIT.java

```
@@ -77,15 +74,21 @@ void testPostAndGetClauseWithExistingDrugMedicationConditions() {
         var clause = clauses.getFirst();
         assertThat(clause)
                 .usingRecursiveComparison()
-                .ignoringFields("uuid")
+                .ignoringFields("uuid", "error")
```

Det er vist ikke nødvendigt at ignore error nu